src/server/saved_objects/client/lib/repository.js

```diff
@@ -0,0 +1,412 @@
+import uuid from 'uuid';
+
+import { getRootType } from '../../../mappings';
+import { getSearchDsl } from './search_dsl';
+import { trimIdPrefix } from './trim_id_prefix';
+import { includedFields } from './included_fields';
+import { decorateEsError } from './decorate_es_error';
+import * as errors from './errors';
+
+export class SavedObjectsRepository {
+  constructor(options) {
```

Comment:
Did you make any changes to the contents of any of these functions when you copied them from the client?

Reply:
This is something that bothers me a little, too. I want to see the change as a rename, but content changes happened, too, like naming changes here and there, so git doesn't detect a rename.

Reply:
No changes were made to the actual implementation, not sure why it's showing up this way in the diff.

Reply:
The code looks fine at a high level, so I really just wanted to know whether I should put extra scrutiny on any particular portion of it. Consider this one resolved for me.

Reply:
(it shows up this way because content changed, and because the old file still exists)